bazel/kotlin_test.bzl

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_test")
+
+# A basic macro to make it easier to declare and run kotlin tests
+#
+# Ergonomic improvements include:
+# 1. Avoiding the need to declare the test_class which requires a fully qualified class name (example below)
+# 2. Avoiding the need to redeclare common unit testing dependencies like JUnit
+# 3. Ability to run more than one test file per target
```

Comment:
If this is too controversial, we can have the sources parameter take a single file rather than a list of files